adm_program/modules/dates/dates.php

```diff
@@ -33,8 +33,8 @@
 unset($_SESSION['dates_request']);
 
 // Initialize and check the parameters
-$disableAdditionalGuests   = 'disabled';
-$disableComments           = 'disabled';
+$disableAdditionalGuests   = (int) 2;
```

Comment:
Forgot to remove. Sorry. 
Needed this for testing with basic html form implementation.
After changing to HtmlForm with property => FIELD_HIDDEN is not nescessary anymore.